code/ship/ship.cpp

```diff
@@ -1700,6 +1704,8 @@ ship_info::ship_info()
 	score = 0;
 
 	scan_time = 2000;
+	scan_range_normal = 150.0f;
+	scan_range_capital = 300.0f;
```

Comment:
Is it correct that these values are the constants defined by `CARGO_REVEAL_MIN_DIST` and `CAP_CARGO_REVEAL_MIN_DIST`? If that is true, maybe it would be better to use those constants here instead to make sure that changing the constants actually has an effect.

Reply:
Yes - I was originally intending to remove the \#define constants since they are no longer required. But I changed my mind after finding commented out code using the constants that I didn't want to change. I could go back & remove the constants as I originally intended, and move the comments with the original comments to this location.

Sorry, I'm not understanding your second sentence. I tested the changes on two missions in MVP 3.8.1 to ensure that the original values would correctly be used if nothing was defined for a ship class.

Reply:
My second sentence was meant to show why the constants should be used (but that is pretty obvious). It doesn't have to do anything with actual gameplay changes. I checked the values as well so there is no problem here.

I'm fine with either solution (removing the constants or using the constants here). If they are only used in code that is commented out then I would suggest simply removing the constants.

Reply:
actually - not important, I'll make the suggested changes & push shortly